CMakeLists.txt.user

```diff
@@ -0,0 +1,878 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE QtCreatorProject>
+<!-- Written by QtCreator 4.5.2, 2018-07-26T10:41:00. -->
+<qtcreator>
```

Comment:
This file should not be checked into git because it is of no value to other users of the code. I suggest to add an exception to `.gitignore` to prevent that this file is pulled into the repository again.